starters/lit-mobx-vaadin/src/router.ts

```diff
@@ -0,0 +1,30 @@
+import { Route, Router } from '@vaadin/router';
+
+import './pages/counter-example/starter-counter.js';
+import './pages/fetch-example/starter-fetch.js';
+import './pages/home/starter-home.js';
+import './pages/starter-layout.js';
+
+const routes: Route[] = [
+  {
+    path: '',
+    component: 'starter-layout',
+    children: [
+      {
+        path: '',
+        component: 'starter-home',
+      },
+      {
+        path: 'counter',
+        component: 'starter-counter',
+      },
+      {
+        path: 'api-example',
```

Comment:
Lets call this `greet` instead - To follow the [spec doc](https://docs.google.com/document/d/1iEXK6BFgu5HZ4dfU7pfv80zLOAd86T8AzstCMEYTUXA/edit#)